### PR TITLE
Sidhaw/schema updates meeting context for static and configurable Tabs

### PR DESCRIPTION
**Pull request by sidhaw**
Sidhaw/schema updates meeting context for static and configurable Tabs

### CODE REVIEW

File: MicrosoftTeams.schema.json [hunk 1]
```diff
@@ -185,6 +185,23 @@
...
           },
+          "context": {
+            "type": "array",
+            "description": "The set of contextItem scopes that a tab belongs to",
+            "maxItems": 8,
+            "items": {
+              "enum": [
+                "personalTab",
+                "channelTab",
+                "privateChatTab",
```
Comment (greybax) [inline MicrosoftTeams.schema.json:196]: looks like these `Enums` are new:
* privateChatTab
* meetingChatTab
* meetingDetailsTab
* meetingSidePanel
* meetingStage
* callingSidePanel

Please make sure that we have support of them in MiddleTier. @Anoop-D fyi

Comment (Anoop-D) [reply MicrosoftTeams.schema.json:196]: @greybax Thanks for checking, this property has already been added in MiddleTier
https://domoreexp.visualstudio.com/Teamspace/_git/Teamspace-MiddleTier/pullrequest/248031?_a=overview

Comment (sidhaw) [reply MicrosoftTeams.schema.json:196]: Yes these enum values are available in Middle Tier

**Comment by sidhaw** [comment]
Tested by creating a manifest with this schema url and set your version to DevPreview
https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/f159d2147c789acec3586febd7d5164dd20b6346/MicrosoftTeams.schema.json

**Pull request merged**